app.js

```diff
@@ -29,9 +27,11 @@ const API_PASSWORD = (process.env.API_PASSWORD) ?
 const API_USER_ID = (process.env.API_USER_ID) ?
   process.env.API_USER_ID : config.get('apiUserId');
 
-if (!(APP_SECRET && VALIDATION_TOKEN && PAGE_ACCESS_TOKEN && API_URI && API_USERNAME && API_PASSWORD && API_USER_ID)) {
-  console.error('Missing config values');
-  process.exit(1);
+if (!(APP_SECRET && VALIDATION_TOKEN && PAGE_ACCESS_TOKEN)) {
+  if (API_URI && API_USERNAME && API_PASSWORD && API_USER_ID) {
```

Comment:
No need to split into 2 `if`. 

BTW, it seems the changed logic is not right. If any value in the second `if` is `null` or `undefined` which results in `false` evaluation, it will not enter the `console.error`.